### PR TITLE
Clarify precedence

### PR DESCRIPTION
**Pull request by Dmitry-Me**
The current code is correct but harder to read than needed - operations precedence is not immediately clear.

These suspicious pieces were found with Cppcheck.

### CODE REVIEW

File: src/md/enc/metamodelrw.cpp [hunk 1]
```diff
@@ -8780,7 +8780,7 @@ bool FilterTable::IsTokenMarked(
         //
         return true;
     }
-    return ( (*Get(rid)) & bitMarked ? true : false);
+    return ( ((*Get(rid)) & bitMarked) ? true : false);
```
Comment (janvorli) [inline src/md/enc/metamodelrw.cpp]: Since you are touching this, could you please also remove the extra space after the first paren?

**Comment by janvorli** [comment]
LGTM

File: src/md/enc/metamodelrw.cpp (hunk 1 above)
Comment (Dmitry-Me) [inline src/md/enc/metamodelrw.cpp]: @janvorli Sure, but I'm kind of worried by DNFBOT and the legal stuff. Why was my first PR accepted without requiring CLA but this one does require CLA?

Comment (janvorli) [inline src/md/enc/metamodelrw.cpp]: @Dmitry-Me I guess that it due to the area you are touching. The previous PR was in PAL, which is Unix / CoreCLR specific. This one touches code that I think is common with full desktop CLR.
@richlander, am I right?

Comment (jkotas) [inline src/md/enc/metamodelrw.cpp]: The trigger for CLA is based on amount of churn, e.g. number of lines changed.

Comment (Dmitry-Me) [inline src/md/enc/metamodelrw.cpp]: @jkotas LOL. How high is the bar?

Comment (jkotas) [inline src/md/enc/metamodelrw.cpp]: @Dmitry-Me The bar is not strictly defined. It is a heuristic e.g. to not require CLA for a trivial typo fix usually.

**Comment by Dmitry-Me** [comment]
Okay, I'll split this into several smaller requests.

**Comment by Eyas** [comment]
@Dmitry-Me wait, why are you splitting into smaller requests? So that you don't have to sign the CLA? Every contributor of significant commits is required to sign a CLA. The PR seems to make sense as-is, dividing it into smaller commits will just unnecessarily bloat the commit history for no reason other than avoiding to sign a CLA. What are your concerns with it?

**Comment by jasonwilliams200OK** [comment]
I think some people when see "legal" and "microsoft" words together, they tend to avoid the offer or consult the lawyer without reading what is written on the agreement; in comparison with what is trending in other repositories etc. 
But this is called freedom I suppose.. and it will take some time for those people to realize that this is not some kind of scheme, it is just another CLA which implicitly says "don't ask us for money for your contribution", which should not be a news for any GitHub contributor anyway!

**Pull request closed without merging**